mcs/class/corlib/Test/System/MathTest.cs

```diff
@@ -405,7 +405,7 @@ public void TestFloor ()
 		public void TestIEEERemainder ()
 		{
 			double a = Math.IEEERemainder (y, x);
-			double b = 0.0050000000000007816;
+			double b = 0.0050000000000010592;
 
```

Comment:
I agree, we are importing reference sources mainly to improve our .net compatibility.

Math.Log (0.1234, double.PositiveInfinity) result is another combination where we returned different result compare to .net.

LGTM